### PR TITLE
docs: add horizontally scale how-to

### PR DESCRIPTION
**Pull request by amandahla**
### Overview

This PR adds a how to about scaling Synapse up.

### Rationale

Horizontal scaling is in place but we don't have a documentation about it yet.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Manging Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: docs/how-to/horizontally-scale.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+the next step.
+
+### Deploy S3-integrator and integrate it with Synapse
+
+This will enable S3 storage provider so media will be stored on a S3 bucket. Replace the
+configuration options with your specific configuration.
+
+```
+juju deploy s3-integrator s3-integrator-media --channel edge
+juju config s3-integrator-media endpoint=http://endpoint bucket=synapse-media-bucket path=/media region=us-east-1 s3-uri-style=path
```
Comment (erinecon) [inline docs/how-to/horizontally-scale.md:36]: For my understanding, are any of the configurations in this command set to dummy values, or would this command work out of the box?

File: docs/how-to/horizontally-scale.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+SAAS             Status       Store         URL
+postgresql       active       cloud1  admin/prod-chat-synapse-db.postgresql
+
+App                       Version  Status   Scale  Charm                     Channel        Rev  Address        Exposed  Message
+media-s3-integrator                active      1  s3-integrator             latest/stable   13  10.10.10.1      no         
+redis                     7.0.4    active       1  redis-k8s                 latest/edge     27  10.10.10.2   no       
+synapse                            active       3  synapse                   latest/stable  303  10.10.10.3    no       
+
+Unit                         Workload     Agent  Address          Ports  Message      
+media-s3-integrator/0*       maintenance  idle   192.168.1.2
```
Comment (erinecon) [inline docs/how-to/horizontally-scale.md:68]: Should the `media-s3-integrator/0` unit be active idle, or is it shown as `maintenance` on purpose?

File: docs/how-to/horizontally-scale.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+With all integrations set, scale Synapse up by running the following command.
+
+```
+juju scale-application synapse 3
+```
+
+### Verify status
+
+The output of `juju status` should look like this now.
```
Comment (erinecon) [inline docs/how-to/horizontally-scale.md:52]: Nitpick: `juju status` --> `juju state --relations`

File: docs/how-to/horizontally-scale.md (hunk 1 above)
Comment (amandahla) [reply docs/how-to/horizontally-scale.md:36]: They are dummy values. In the line before, there is a guidance regarding that "Replace the
configuration options with your specific configuration."  Similar to existing one
https://discourse.charmhub.io/t/synapse-docs-configure-smtp/13210 for example.

File: docs/how-to/horizontally-scale.md (hunk 2 above)
Comment (amandahla) [reply docs/how-to/horizontally-scale.md:68]: Ops, it should be active, thanks for noticing!

File: docs/how-to/horizontally-scale.md (hunk 1 above)
Comment (amandahla) [reply docs/how-to/horizontally-scale.md:40]: Thanks for raising this. I tried to put more emphasis in that now.

Comment (erinecon) [reply docs/how-to/horizontally-scale.md:40]: Thanks for the explanation and edit. I think it's clearer now :)

**Review by erinecon — APPROVED** [review]
LGTM

**Review by erinecon — APPROVED** [review]

**Review by javierdelapuente — APPROVED** [review]

**Pull request merged**